content/about/myths.md

```diff
@@ -21,6 +21,37 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
 
 But even ignoring the high performance, specialist end, XML still holds its own. Consider [these findings](https://www.balisage.net/Proceedings/vol10/html/Lee01/BalisageVol10-Lee01.html), for example.
 
+## XMPP is not secure.
+
+The hypothesis:
+> XMPP is not secure and does not implement modern technology.
+
+The fact:
+> XMPP offers state-of-the-art end-to-end encryption, and a relatively high-degree of pseudonymity
+
+However, we do not ever claim that XMPP is secure. "Secure" is not an absolute. In order to decide whether XMPP is secure, you need to decide what you want to be secure against. No system that relies on servers is secure against for example a rogue administrator (end-to-end encryption can protect your message contents even against a rogue admin, but only if you verify keys appropriately). Instead, our specifications consistently remind developers of the risks of various choices. Plenty of XMPP extensions will certainly 
```

Comment:
> However, we do not ever claim that XMPP is secure.

I would remove this sentence entirely, it's not necessary, is overly defensive, and looks as if you're admitting that XMPP is not secure.